docs/rtd-requirements.txt

```diff
@@ -32,13 +32,11 @@ django-widget-tweaks==1.4.8
 djangorestframework==3.12.4
 docutils==0.16
 Fabric3==1.14.post1
-flower==0.9.7
```

Comment:
Why is this removed?

Reply:
[This](https://github.com/mher/flower/issues/1029#issuecomment-835486909) seems to suggest that we might have to modify our production setup slightly, but `flower` + `celery` should still work.

Reply:
It looks like there hasn't actually been a new build cut that supports celery 5.0:
https://github.com/mher/flower/issues/1109
So it looks like it makes sense to either wait for that to be resolved or avoid bumping celery in the meantime.
That said, I can't actually see where flower is used in ion...

Reply:
I'm inclined to wait. This looks like a pretty active area of discussion and `flower` seems to be maintained. We use `flower` in the production setup for monitoring `celery`. I'm going to leave this PR open as a reminder to check back for a new `flower` release.

Reply:
SGTM, I'll sit on this for now.